uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/PromptHandlerParameters.java

```diff
@@ -44,23 +44,43 @@
     @NonNull
     private final PromptParameter prompt;
 
+    // the login hint that was provided
+    private final String loginHint;
+
     // whether session is expected or not (via presence of a cookie)
     private final boolean sessionExpected;
 
-    // whether login hint was supplied or not to the interactive request
-    private final boolean loginHintProvided;
+    // The broker that should be being used for this request
+    private final ITestBroker broker;
+
+    // whether we expected the broker account chooser activity to appear
```

Comment:
The location of these comments is strange.  These are all private and final.  Do these single-line comments feed some javadoc generation via Lombok?  I thought that was required to be javadoc to start with.

Reply:
Probably not, I'll change these to javadoc

Reply:
Addressed